src/providers/test-submission/test-submission.ts

```diff
@@ -69,5 +81,8 @@ export class TestSubmissionProvider {
     };
     return removeNullFields(data);
   }
-
+  removeFieldsForPartialData = (data: StandardCarTestCATBSchema): Partial<StandardCarTestCATBSchema> => {
+    data.testSummary = null;
```

Comment:
We can't do this as this will remove `D255`, `debriefWitnessed` from the state which aren't on the office page